.github/workflows/points.yml

```diff
@@ -0,0 +1,53 @@
+name: Points
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  build-rails-current:
+    runs-on: ubuntu-latest
+    
+    env:
+      POSTGRES_USER: postgres
```

Comment:
I think that we can reuse the configuration from the previous job

if we do `env: &ENV` when we define the first one then we can do

```
env:
  << *ENV
```

here

the sam can be done for the postgres config I think (I'm not 100% sure if that works between jobs, but it's worth to try)